recipes/thesias/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "3.1" %}
+
+package:
+  name: thesias
+  version: {{ version }}
+
+source:
+  url: https://github.com/daissi/thesias/archive/bd558973415a3ab13e24df1204cc8443858318e8.tar.gz
```

Comment:
Can we use the tagged 3.1 release instead?

Reply:
The tagged 3.1 is only to provide an old (2007) archive with only binaries. I will release something like 3.1.1 as soon as I have fixed the makefile. It's why I am packaging THESIAS for bioconda and Debian to seen if everything is fine on different systems. For now, it is better to use git master with maybe something like 3.1.post1 if you prefer.

Reply:
Why not just tag 3.1.1 now then? Our bot will then automatically create new PR when you tag new releases.

Reply:
I have tagged 3.1.1 and updated the recipe accordingly.